### ISSUE TITLE
Properties don't report "get" vs. "set"

### ISSUE THREAD

**Issue by stephencelis** [realm/jazzy#225]
It would be great for properties to be given the same treatment they're given in the module interfaces, _e.g._,

``` swift
public var completed: Bool { get }
public var enabled: Bool { get set }
```

**Comment by jpsim** [comment]
Agreed! I think the solution here might be to use SourceKit's declaration, which does this automatically. The reason we haven't done that so far is that 1) it doesn't show default argument values (instead showing `arg: T = default`) and 2) it always uses full type names `Swift.String`/`Foundation.NSDate`.

It might be a bit tricky to solve both issues, but I think it's worthwhile.

**Comment by stephencelis** [comment]
Updated to clarify that this impacts non-computed properties, too.

**Comment by jpsim** [comment]
just an FYI, you get this for the Swift declarations for Objective-C docs now, since those use the generated Swift interface.

**Comment by istx25** [comment]
Seems resolved to me.

>just an FYI, you get this for the Swift declarations for Objective-C docs now, since those use the generated Swift interface.

Feel free to re-open if you disagree or if more needs to be done.

**Comment by jpsim** [comment]
Reopening since it'd still be useful to do for Swift properties.

**Comment by johnfairh** [comment]
Fixed via #902.
